server/utils/commonUtils.js

```diff
@@ -0,0 +1,10 @@
+const crypto = require('crypto');
+
+function hashPassword(password) {
+    const secret = "I'm a teapot" || process.env.HASH_SALT;
```

Comment:
If you want this env var (HASH_SALT) to be used, it needs to be to the left of the or operator.  Currently, since "I'm a teapot" is always truthy, it will always be used as the secret.

Reply:
Oh cool. Thanks @wallacepreston. 
That one slipped my mind.